flash/vision/classification/data.py

```diff
@@ -56,7 +56,7 @@ def __init__(
         self.transform = transform
         self.loader = loader
         if self.has_labels:
-            self.label_to_class_mapping = {v: k for k, v in enumerate(list(sorted(list(set(self.fnames)))))}
+            self.label_to_class_mapping = {v: k for k, v in enumerate(list(sorted(list(set(self.labels)))))}
```

Comment:
```suggestion
            self.label_to_class_mapping = {v: k for k, v in enumerate(sorted(set(self.labels)))}
```
maybe we can just zip

Reply:
I have updated the code. I think it now looks cleaner.